src/app/_components/_sharedcomponents/CreateGameForm/CreateGameForm.tsx

```diff
@@ -60,7 +61,7 @@ const CreateGameForm: React.FC<CreateGameFormProps> = ({
 		useState<string>("Vader Green Ramp");
 	const [deckLink, setDeckLink] = useState<string>("");
 	const [saveDeck, setSaveDeck] = useState<boolean>(false);
-	const [deckData, setDeckData] = useState<DeckData | null>(null);
+	//let [deckData, setDeckData] = useState<DeckData | null>(null); Because of Async this won't set in the correct timeframe
```

Comment:
This was just a temporary holding spot for the data so we can get rid of this for now since the form has no reason to hold it.